### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by spalger**
- trim down basic usage example, which should be sufficient for most uses
- describe settings and when to use each in a table and then a single annotated code snippet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,58 @@
-# eslint-import-resolver-kibana
+# @elastic/eslint-import-resolver-kibana
 
 Resolver for Kibana imports, meant to be used with [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import).
 
 ## Usage
 
-In your `.eslintrc.(yml|json|js)` file, add the following `import/resolver` under `settings`:
+Specify this resolver at `settings["import/resolver']` in your eslint config file:
```
Comment (w33ble) [inline README.md]: Implies js format rc file, no?

File: README.md [hunk 2]
```diff
@@ -1,43 +1,58 @@
...
-  # uses 'eslint-import-resolver-kibana':
-  import/resolver:
-    - "@elastic/eslint-import-resolver-kibana"
+  import/resolver: "@elastic/eslint-import-resolver-kibana"
 ```
 
-The `node` resolver isn't strictly required, but the resolving rules are unlikely to work as you expect if you don't include it in the list of resolvers. It's only included by default if you don't specify other resolvers.
+## Settings
+
+> NOTE: all relative paths are resolved as relative to the project root, which is determined by walking up from the first linted file and looking for a `package.json` file. If you project has multiple `package.json` files then make sure to specify the `rootPackageName` setting.
```
Comment (w33ble) [inline README.md]: Remove the blockquote, it makes the text easy to miss.

> If you project has multiple `package.json` files then make sure to specify the `rootPackageName` setting.

I would keep this in the description. I personally tend to skim over surrounding text when I see descriptions laid out individually. First thing I thought when I looked at this readme was "Where is that warning I added before? Seem important..."

File: README.md [hunk 3]
```diff
@@ -1,43 +1,58 @@
...
+
+Property | Default | Descritpion
```
Comment (w33ble) [inline README.md]: sp: Description

File: README.md [hunk 4]
```diff
@@ -1,43 +1,58 @@
...
+-------- | ------- | -----------
+rootPackageName | `null` | The `"name"` property of the root `package.json` file
+kibanaPath | `../kibana` | Relative path to the kibana root
+pluginDirs | `[]` | Array of relative paths pointing to directories which contain Kibana plugins
```
Comment (w33ble) [inline README.md]: Add a note that each plugin requires its own `package.json` file to be included.

**Review by w33ble — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (spalger) [reply README.md]: not necessarily, I'm really just trying to describe the property. The copiable code snippet is below, but I can use a more prose version.

**Review by w33ble — APPROVED** [review]

**Pull request merged**